packages/runtime/lib/src/build.dart

```diff
@@ -122,6 +123,26 @@ class Build {
     }
   }
 
+  Future<void> searchOldPackageUsage(Directory targetDir) async {
+    final files = targetDir
+        .listSync(recursive: true)
+        .whereType<File>()
+        .where((element) => element.path.endsWith(".dart"));
+    for (final file in files) {
+      final content = await file.readAsString();
+      if (content.contains("package:conduit/conduit.dart")) {
+        final newContent = content.replaceAll(
+          "package:conduit/conduit.dart",
```

Comment:
This replace will fail on newer version because there will be two imports for conduit.